src/ripple/app/consensus/RCLValidations.h

```diff
@@ -25,12 +25,16 @@
 #include <ripple/protocol/Protocol.h>
 #include <ripple/protocol/RippleLedgerHash.h>
 #include <ripple/protocol/STValidation.h>
+#include <optional>
+#include <set>
 #include <vector>
 
 namespace ripple {
 
 class Application;
 
+enum class BypassAccept { FALSE, TRUE };
```

Comment:
The `FALSE` and `TRUE` keywords are `#define`d on Windows, which messes up building this.
```suggestion
enum class BypassAccept : bool { no = false, yes };
```

Reply:
I pushed this change to the branch so I can go ahead and merge this pull request. and let testing proceed.